src/main/java/forestry/arboriculture/models/SimpleModel.java

```diff
@@ -154,7 +152,7 @@ public IModel retexture(ImmutableMap<String, String> textures) {
 				try {
 					model = ModelLoaderRegistry.getModel(loc);
 				} catch (Exception e) {
-					throw Throwables.propagate(e);
+					throw new RuntimeException(e);
```

Comment:
this should be
```java
Throwables.throwIfUnchecked(e);
throw new RuntimeException(e);
```
That way the exception is only wrapped if it needs to be.

You can find out more about it here: https://github.com/google/guava/wiki/Why-we-deprecated-Throwables.propagate